code/script.js

```diff
@@ -1,53 +1,153 @@
 // DOM selectors (variables that point to selected DOM elements) goes here 👇
-const chat = document.getElementById('chat')
-
+const chat = document.getElementById("chat");
+const input = document.getElementById("name-input");
+const form = document.getElementById("name-form");
+const sendSound = new Audio("assets/send-message.mp3");
+const receiveSound = new Audio("assets/receiving.mp3");
+let values = {};
 // Functions goes here 👇
 
 // A function that will add a chat bubble in the correct place based on who the sender is
 const showMessage = (message, sender) => {
   // The if statement checks if the sender is the user and if that's the case it inserts
   // an HTML section inside the chat with the posted message from the user
-  if (sender === 'user') {
+  if (sender === "user") {
+    sendSound.play();
+    console.log("Message handled by user:", message);
```

Comment:
Remove console.logs, they are used for development and shouldn't be left in production